README.md

```diff
@@ -108,6 +108,42 @@ java -cp ./dbeam-core/target/dbeam-core-shaded.jar \
   --skipPartitionCheck
 ```
 
+### Password configuration
+
+Database password can be configured by simply passing `--password=writepasswordhere`, `--passwordFile=/path/to/file/containing/password` or `--passwordFile=gs://gcs-bucket/path/to/file/containing/password`.
+
+A more robust configuration is to point to a [Google KMS](https://cloud.google.com/kms/) encrypted file.
+DBeam will try to decrypt using KMS if the file ends with `.encrypted` (e.g. `--passwordFile=gs://gcs-bucket/path/to/db-password.encrypted`).
```

Comment:
Maybe less automagic by introducing `--passwordFileEncrypted`? Not sure, up to you to decide.

Reply:
Good idea!